src/database/migrations/1669226921155-BaseEnitity.ts

```diff
@@ -0,0 +1,15 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class BaseEnitity1669226921155 implements MigrationInterface {
```

Comment:
This isn't a great migration name, it's not really clear what it's doing. Something Like CreateUsers is preferred.